### PR TITLE
Schedule for the 1.23 release

### PR DESCRIPTION
**Pull request by reylejano**
#### What type of PR is this:
/kind documentation

#### What this PR does / why we need it:
This PR sets the schedule for the 1.23 release to facilitate discussion and gather feedback.

#### Which issue(s) this PR fixes:
None

#### Special notes for your reviewer:
None

/area release-team
/sig release
/assign @justaugustus @saschagrunert @jeremyrickard @cpanato @puerco @hasheddan)
/cc @salaxander @encodeflush @voigt @jlbutler @cici37 @karenhchu 

/hold for reviews

### CODE REVIEW

File: releases/release-1.23/README.md [hunk 1]
```diff
@@ -37,51 +37,52 @@ description: |
...
+| KubeCon NA + Co-located events | | October 11-15, 2021 | week 8 | |
 | 1.23.0-beta.0 released | Branch Manager ([TBD](TBD)) | TBD | TBD | |
-| **Begin [Burndown]** (MWF meetings) | Lead | TBD | TBD | [1.23-blocking], [master-blocking], [master-informing] |
-| **Call for [Exceptions][Exception]** | Lead | TBD | TBD | |
-| Brace Yourself, Code Freeze is Coming | Comms / Bug Triage | TBD | TBD | |
-| **Begin Feature blog freeze** (23:59 PDT) | Comms Lead | TBD | TBD | |
+| **Begin [Burndown]** (MWF meetings) | Lead | Mon November 1, 2021 | week 11 | [1.23-blocking], [master-blocking], [master-informing] |
+| **Call for [Exceptions][Exception]** | Lead | Mon November 1, 2021 | week 11 | |
+| Brace Yourself, Code Freeze is Coming | Comms / Bug Triage | Mon November 1, 2021 | week 11 | |
+| **Begin Feature blog freeze** (23:59 PDT) | Comms Lead | Tues November 2, 2021 | week 11 | |
```
Comment (saschagrunert) [inline releases/release-1.23/README.md:66]: Ultra small nit, let's stick to 3 character abbreviations for days:
```suggestion
| **Begin Feature blog freeze** (23:59 PDT) | Comms Lead | Tue November 2, 2021 | week 11 | |
```

File: releases/release-1.23/README.md [hunk 2]
```diff
@@ -37,51 +37,52 @@ description: |
...
-| Burndown Meetings daily| Lead | TBD | TBD | |
+| **Begin [Code Freeze]** (18:00 PST) | Branch Manager | Thur November 11, 2021 | week 12 | |
+| Docs deadline - Open placeholder PRs | Docs Lead |Fri November 12, 2021 | week 12 | |
+| Burndown Meetings daily| Lead | Mon November 15, 2021 | week 13 | |
 | 1.23.0-beta.2 released | Branch Manager ([TBD](TBD)) | TBD | TBD | |
-| **[Test Freeze]** (18:00 PDT) | Branch Manager | TBD | TBD | |
-| Docs deadline - PRs ready for review | Docs Lead | TBD | TBD | |
+| **[Test Freeze]** (18:00 PST) | Branch Manager | Thur November 18, 2021 | week 13 | |
+| Docs deadline - PRs ready for review | Docs Lead | Tues November 23, 2021 | week 14 | |
 | 1.23.0-rc.0 released | Branch Manager ([TBD](TBD)) | TBD | TBD | |
```
Comment (saschagrunert) [inline releases/release-1.23/README.md:74]: I'd propose to couple **Test Freeze** and the first RC tightly together:
```suggestion
| **[Test Freeze]** (18:00 PST) | Branch Manager | Tue November 23, 2021 | week 14 | |
| Docs deadline - PRs ready for review | Docs Lead | Tue November 23, 2021 | week 14 | |
| 1.23.0-rc.0 released | Branch Manager ([TBD](TBD)) | Tue November 23, 2021 | week 14 | |
```

File: releases/release-1.23/README.md [hunk 3]
```diff
@@ -37,51 +37,52 @@ description: |
...
+| 1.23.0-alpha.1 released | Branch Manager ([TBD](TBD)) | Wed August 25, 2021 | week 1 | |
+| Schedule finalized | Lead | Thur August 26, 2021 | week 1 | |
+| Team finalized | Lead | Fri August 27, 2021 | week 1 | |
+| Start Release Notes Draft | Release Notes Lead | Tues September 7, 2021 | week 3 | |
+| **Begin [Enhancements Freeze]** (23:59 PDT) | Enhancements Lead | Thur September 9, 2021 | week 3 | [master-blocking], [master-informing] |
 | 1.23.0-alpha.2 released | Branch Manager ([@TBD](TBD)) | TBD | TBD | |
 | 1.23.0-alpha.3 released | Branch Manager ([TBD](TBD)) | TBD | TBD | |
-| KubeCon NA + Co-located events | | Oct 11-15 2021 | | Week 8 |
+| KubeCon NA + Co-located events | | October 11-15, 2021 | week 8 | |
 | 1.23.0-beta.0 released | Branch Manager ([TBD](TBD)) | TBD | TBD | |
```
Comment (saschagrunert) [inline releases/release-1.23/README.md:62]: Proposal: Distinguish alpha and beta releases by **Code Freeze**. means, 
```suggestion
| 1.23.0-alpha.4 released | Branch Manager ([TBD](TBD)) | TBD | TBD | |
```

File: releases/release-1.23/README.md [hunk 4]
```diff
@@ -37,51 +37,52 @@ description: |
...
-| **Begin Feature blog freeze** (23:59 PDT) | Comms Lead | TBD | TBD | |
+| **Begin [Burndown]** (MWF meetings) | Lead | Mon November 1, 2021 | week 11 | [1.23-blocking], [master-blocking], [master-informing] |
+| **Call for [Exceptions][Exception]** | Lead | Mon November 1, 2021 | week 11 | |
+| Brace Yourself, Code Freeze is Coming | Comms / Bug Triage | Mon November 1, 2021 | week 11 | |
+| **Begin Feature blog freeze** (23:59 PDT) | Comms Lead | Tues November 2, 2021 | week 11 | |
 | 1.23.0-beta.1 released | Branch Manager ([TBD](TBD)) | TBD | TBD | |
-| **Begin [Code Freeze]** (18:00 PDT) | Branch Manager | TBD | TBD | |
-| Docs deadline - Open placeholder PRs | Docs Lead |TBD | TBD | |
-| Burndown Meetings daily| Lead | TBD | TBD | |
+| **Begin [Code Freeze]** (18:00 PST) | Branch Manager | Thur November 11, 2021 | week 12 | |
```
Comment (saschagrunert) [inline releases/release-1.23/README.md:68]: If we agree having an alpha.4, then beta.0 could align with Code Freeze and being moved to a Tuesday:

```suggestion
| **Begin [Code Freeze]** (18:00 PST) | Branch Manager | Tue November 16, 2021 | week 13 | |
```

File: releases/release-1.23/README.md (hunk 1 above)
Comment (reylejano) [reply releases/release-1.23/README.md]: Schedule is updated with 3 character day abbreviations

File: releases/release-1.23/README.md (hunk 4 above)
Comment (reylejano) [reply releases/release-1.23/README.md]: Updated the schedule with Code Freeze as the demarcation between alpha and beta releases

**Comment by reylejano** [comment]
@saschagrunert , the schedule is updated based on your comments and suggestions, ptal

File: releases/release-1.23/README.md [hunk 5]
```diff
@@ -37,51 +37,52 @@ description: |
...
-| KubeCon NA + Co-located events | | Oct 11-15 2021 | | Week 8 |
+| KubeCon NA + Co-located events | | October 11-15, 2021 | week 8 | |
+| 1.23.0-alpha.4 released | Branch Manager ([TBD](TBD)) | TBD | TBD | |
+| **Begin [Burndown]** (MWF meetings) | Lead | Mon November 1, 2021 | week 11 | [1.23-blocking], [master-blocking], [master-informing] |
+| **Call for [Exceptions][Exception]** | Lead | Mon November 1, 2021 | week 11 | |
+| Brace Yourself, Code Freeze is Coming | Comms / Bug Triage | Mon November 1, 2021 | week 11 | |
+| **Begin Feature blog freeze** (23:59 PDT) | Comms Lead | Tue November 2, 2021 | week 11 | |
+| Burndown Meetings daily| Lead | Mon November 15, 2021 | week 13 | |
+| **Begin [Code Freeze]** (18:00 PST) | Branch Manager | Tue November 16, 2021 | week 13 | |
 | 1.23.0-beta.0 released | Branch Manager ([TBD](TBD)) | TBD | TBD | |
```
Comment (saschagrunert) [inline releases/release-1.23/README.md:69]:
```suggestion
| 1.23.0-beta.0 released | Branch Manager ([TBD](TBD)) | Tue November 16, 2021 | week 13 | |
```

File: releases/release-1.23/README.md [hunk 6]
```diff
@@ -37,51 +37,52 @@ description: |
...
-| **Begin [Burndown]** (MWF meetings) | Lead | TBD | TBD | [1.23-blocking], [master-blocking], [master-informing] |
-| **Call for [Exceptions][Exception]** | Lead | TBD | TBD | |
-| Brace Yourself, Code Freeze is Coming | Comms / Bug Triage | TBD | TBD | |
-| **Begin Feature blog freeze** (23:59 PDT) | Comms Lead | TBD | TBD | |
+| Docs deadline - Open placeholder PRs | Docs Lead | Thu November 18, 2021 | week 13 | |
 | 1.23.0-beta.1 released | Branch Manager ([TBD](TBD)) | TBD | TBD | |
```
Comment (saschagrunert) [inline releases/release-1.23/README.md:71]: @puerco we discussed that today, do you think we can drop beta.1?

File: releases/release-1.23/README.md [hunk 7]
```diff
@@ -37,51 +37,52 @@ description: |
...
-| Burndown Meetings daily| Lead | TBD | TBD | |
-| 1.23.0-beta.2 released | Branch Manager ([TBD](TBD)) | TBD | TBD | |
-| **[Test Freeze]** (18:00 PDT) | Branch Manager | TBD | TBD | |
-| Docs deadline - PRs ready for review | Docs Lead | TBD | TBD | |
-| 1.23.0-rc.0 released | Branch Manager ([TBD](TBD)) | TBD | TBD | |
+| **[Test Freeze]** (18:00 PST) | Branch Manager | Tue November 23, 2021 | week 14 | |
+| Docs deadline - PRs ready for review | Docs Lead | Tue November 23, 2021 | week 14 | |
+| 1.23.0-rc.0 released | Branch Manager ([TBD](TBD)) | Tue November 23, 2021 | week 14 | |
 | release-1.23 branch created | Branch Manager | TBD | TBD | |
 | release-1.23 jobs created | Branch Manager | TBD | TBD | |
```
Comment (saschagrunert) [inline releases/release-1.23/README.md:76]: Happens in one step:
```suggestion
| release-1.23 branch created | Branch Manager | Tue November 23, 2021 | week 14 | |
| release-1.23 jobs created | Branch Manager | Tue November 23, 2021 | week 14 | |
```

File: releases/release-1.23/README.md (hunk 6 above)
Comment (saschagrunert) [reply releases/release-1.23/README.md:71]: @reylejano There were folks who requested a beta.1 in recent releases. In theory we would not have much time within 1 week between beta.0 and rc.0, so let's drop that line for now.

File: releases/release-1.23/README.md [hunk 8]
```diff
@@ -37,51 +37,52 @@ description: |
...
 |---|---|-------|---|---|
-| Start of Release Cycle | Lead | Mon August 23rd 2021 | week 1 | [master-blocking] |
-| Start Enhancements Tracking | Enhancements Lead | Mon August 23rd 2021| Week 1 | |
-| 1.23.0-alpha.1 released | Branch Manager ([TBD](TBD)) | Wed August 25 2021 | Week 1 | |
-| Schedule finalized | Lead | Thur August 26 2021 | week 1 | |
-| Team finalized | Lead | Fri August 27 2021 | week 1 | |
-| **Begin [Enhancements Freeze]** (23:59 PDT) | Enhancements Lead | TBD | TBD | [master-blocking], [master-informing] |
+| Start of Release Cycle | Lead | Mon August 23, 2021 | week 1 | [master-blocking] |
+| Start Enhancements Tracking | Enhancements Lead | Mon August 23, 2021| week 1 | |
+| 1.23.0-alpha.1 released | Branch Manager ([TBD](TBD)) | Wed August 25, 2021 | week 1 | |
```
Comment (saschagrunert) [inline releases/release-1.23/README.md:54]: Let's remove all those `TBD`s for the branch manager, the links go into nowhere and we do not have mentioned personas anywhere else in that doc:
```suggestion
| 1.23.0-alpha.1 released | Branch Manager | Wed August 25, 2021 | week 1 | |
```

**Review by saschagrunert — COMMENTED** [review]
Just a few comments left, thanks for addressing them @reylejano.

**Comment by reylejano** [comment]
@saschagrunert , removed beta.1 and the TBD links for the Branch Manager and updated the 1.23 branch & jobs creation dates, ptal

**Review by saschagrunert — APPROVED** [review]
/lgtm

**Pull request merged**